vips/vip-converter/vip-converter-testnet/vip-converter.ts

```diff
@@ -0,0 +1,205 @@
+import { ProposalType } from "../../../src/types";
+import { makeProposal } from "../../../src/utils";
+import {
+  ACM,
+  BTCBPrimeConverterTokenOuts,
+  BTCB_PRIME_CONVERTER,
+  BaseAssets,
+  CONVERTER_NETWORK,
+  ETHPrimeConverterTokenOuts,
+  ETH_PRIME_CONVERTER,
+  NORMAL_TIMELOCK,
+  RISK_FUND_CONVERTER,
+  RiskFundConverterTokenOuts,
+  USDCPrimeConverterTokenOuts,
+  USDC_PRIME_CONVERTER,
+  USDTPrimeConverterTokenOuts,
+  USDT_PRIME_CONVERTER,
+  XVSVaultConverterTokenOuts,
+  XVS_VAULT_CONVERTER,
+} from "./Addresses";
+import {
+  acceptOwnershipCommandsAllConverters,
+  addConverterNetworkCommandsAllConverters,
+  addTokenConverterCommandsAllConverters,
+  callPermissionCommandsAllConverter,
+  incentiveAndAccessibilityForBTCBPrimeConverter,
+  incentiveAndAccessibilityForETHPrimeConverter,
+  incentiveAndAccessibilityForRiskFundConverter,
+  incentiveAndAccessibilityForUSDCPrimeConverter,
+  incentiveAndAccessibilityForUSDTPrimeConverter,
+  incentiveAndAccessibilityForXVSVaultConverter,
+} from "./commands";
+
+const DEFAULT_PROXY_ADMIN = "0x7877fFd62649b6A1557B55D4c20fcBaB17344C91";
+const RISK_FUND_PROXY = "0x487CeF72dacABD7E12e633bb3B63815a386f7012";
+const RISK_FUND_V2_IMPLEMENTATION = "0x6b925876F9e007b7CD0d7EFd100991F3eF4a4276";
+
+const PROTOCOL_SHARE_RESERVE_PROXY = "0x25c7c7D6Bf710949fD7f03364E9BA19a1b3c10E3";
+const PROTOCOL_SHARE_RESERVE_NEW_IMPLEMENTATION = "0xEdaB2b65fD3413d89b6D2a3AeB61E0c9eECA6A76";
+
+const VTREASURY = "0x8b293600C50D6fbdc6Ed4251cc75ECe29880276f";
+const XVS_VAULT_TREASURY = "0xab79995b1154433C9652393B7BF3aeb65C2573Bd";
+
+export const vipConverter = () => {
+  const meta = {
+    version: "v2",
+    title:
+      "VIP-converter1 Upgrades the implementation of RiskFund and ProtocolShareReserve with Adding of converts in ConverterNetwork and vice versa. It also sets conversion configs for the converters",
+    description: `
+    Upgrade the implementation of riskfund to riskfund V2
+    sets RiskFundConverter in RiskFundV2
+    Upgrade the implementation of ProtocolShareReserve and update the distributionConfigs
+    update destination address to Riskfund in RiskFundConverter
+    Add Converters in ConverterNetwork
+    Add ConverterNetwork in Converters
+    Sets Conversion configs for the following converters:-
+    1. RiskFundConverter
+    2. USDTPrimeConverter
+    3. USDCPrimeConverter
+    4. BTCBPrimeConverter
+    5. ETHPrimeConverter
+    6. XVSVaultConverter`,
+
+    forDescription: "Execute this proposal",
+    againstDescription: "Do not execute this proposal",
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      ...acceptOwnershipCommandsAllConverters,
+      {
+        target: XVS_VAULT_TREASURY,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: CONVERTER_NETWORK,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+
+      ...callPermissionCommandsAllConverter,
```

Comment:
Fast-track and Critical timelocks should be also granted

Reply:
Done https://github.com/VenusProtocol/vips/pull/49/commits/0473a2acadf59f75370bf39aa959d6cd9a55476e